comfy_cli/command/run.py

```
@@ -35,6 +38,45 @@ def load_api_workflow(file: str):
+
+def convert_ui_workflow_via_server(workflow: dict, host: str, port: int, timeout: int) -> dict:
+    """POST a UI-format workflow to the server's /workflow/convert and return API-format JSON.
+
+    Raises WorkflowConverterUnavailable if the server doesn't expose the endpoint.
+    Raises typer.Exit on other conversion failures.
+    """
+    url = f"http://{host}:{port}/workflow/convert"
+    req = request.Request(url, json.dumps(workflow).encode("utf-8"))
+    req.add_header("Content-Type", "application/json")
+    try:
+        resp = request.urlopen(req, timeout=timeout)
```

Pylint (4.0.5)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)